apps/challenges/aws_utils.py

```diff
@@ -531,6 +531,8 @@ def create_service_by_challenge_pk(client, challenge, client_token):
 
     queue_name = challenge.queue
     service_name = "{}_service".format(queue_name)
```

Comment:
@Ram81 we get the service name from here and then pass this in the `create_service` boto api call

Reply:
@Ram81 yeah makes sense. I can add a utils function for this.

Reply:
I have fixed the build fail and saved the service name from api response for now after the latest change. If you say, I would revert to the previous method and make a utils function as well :smile: